index.html

```diff
@@ -301,6 +301,55 @@ <h3>
           </table>
         </section>
 
+        <section id="normative">
+          <h3>
+            Registries
+          </h3>
+          <p>
+A series of at least the following registries that support the normative
+deliverables of the Working Group will be published and maintained:
+          </p>
+          <table>
+            <tr>
+              <th>Registry</th>
+              <th>Description</th>
+            </tr>
+            <tr>
+              <td>
+<a href="https://w3c-ccg.github.io/vc-extension-registry/">Verifiable Credential Extensions Registry</a>
```

Comment:
I agree with this registry. However, I suggest we remove a link pointing to a CCG work item since a new registry will be created and having this link is misleading.

Reply:
Personally I find these links useful to frame what the registry will likely look like, however we could provide a note to clarify that these linked registries are not finalised and merely included as informative links?

Reply:
I really like the `Based on` column and would agree if the links are moved there.

Reply:
What @TallTed said, and fixed in d38c69b9e1e132b16fd05b14e0e74bba88f316e9.